templates/service-template.yml

```diff
@@ -921,7 +950,10 @@ objects:
               subPath: dynamic-scaling-configuration.yaml
             - name: kas-fleet-manager-node-prewarming-config
               mountPath: /config/node-prewarming-configuration.yaml
-              subPath: node-prewarming-configuration.yaml  
+              subPath: node-prewarming-configuration.yaml
+            - name: kas-fleet-manager-admin-authz-config
+              mountPath: config/admin-authz-configuration.yaml
```

Comment:
```suggestion
              mountPath: /config/admin-authz-configuration.yaml
```

Reply:
All the configs are located in the root `/config`.
This suggestion adds a missing `/`